website/src/content/blog/roadmap-2024.md

```diff
@@ -0,0 +1,360 @@
+- Offer guides on our website to users who want to migrate from ESlint (CLI commands and configuration)
+- Offer a section on our website that shows a mapping of the ESLint rules to our rules
+- Offer commands to ease the transition
+  - A command called `biome migrate prettier` that will read `.prettierrc` and `.prettierignore`, will update the `biome.json` file (or create it) with the configuration coming from the Prettier files.
+  - A command called `biome migrate eslint` that will read the JSON configuration of Eslint and the ignore file. There will be expectations and limitations.
+
+
+### Expand Biome's language support
+
+CSS is our next language of focus and is making good progress. HTML and Markdown will follow. Follow our [up-to-date page](/internals/language-support) to keep up with the progress of our work.
+
+The CSS language will enable a lot of work and experimentation: CSS formatting and linting, and we will port some of the lint rules from `styelelint`. A new area of experimentation is cross-linting.
+
+The idea of cross-linting can be explained with an example: compute the CSS styles/classes defined in a project, and warn a user when said styles aren't used inside JSX/HTML files.
+
+Plus, we unlock another area of experimentation, which is embedded formatting.
+
+HTML and Markdown will be our next languages of focus. HTML will enable us to parse other variants of HTML that are popular in the frontend ecosystem: [Vue](https://vuejs.org/), [Svelte](https://svelte.dev/) and [Astro](https://astro.build/). This would require some exploration on how to represent super languages of HTML.
+
+### Deepen Biome's existing capabilities to offer more functionalities.
+
+- Project analysis and dependency resolution
+- Type system
+- CLI
+
+#### Project analysis and dependency resolution
+
+We will provide lint rules that can read the manifest and do operations over it, such as invalid licenses.
+
+With project resolution, we will be able to provide more lint rules, some of which will be able to detect unused modules.
+
+With dependency resolution, we will be able to provide to - for example - detect dependencies that aren't used inside a project.
+
+With this infrastructure, our LSP is going to be more powerful and provide more features, for example:
+- rename variables across a project;
+- auto-complete for imports;
+- in-line types
+
+#### Type system
+
+Building a full-fledged type system such as TypeScript is a massive effort, that's why we decided to take a different direction and start by building a subset of the type system that would require stricter typing. This approach would allow us to build some important lint rules that users have been asking for.
+
+This will come with a downside: relying on a stricter code, and minimal type inference from the compiler.
+
+Once we have something we can rely on, we can slowly widen the type-checking functionality.
```

Comment:
I think we should avoid using the expression `type-cheking` because we are not implementing a type checker.

Maybe:

```suggestion
Once we have something we can rely on, we can slowly widen the capabilities of our type system.
```